### ISSUE TITLE
Convert some elements that are not `keyboard focusable` to be` keyboard focusable` as they should be

### ISSUE THREAD

**Issue by mithi** [mithi/epic-react-exercises#30]
<img width="641" alt="Screen Shot 2021-01-26 at 5 05 57 AM" src="https://user-images.githubusercontent.com/1670421/105766177-62a96800-5f94-11eb-8c59-8ad8e8806163.png">
